### PR TITLE
vendor containerd v0.0.0-20210721160646-ee27cde735e2

### PR DESCRIPTION
**Pull request by crazy-max**
test containerd update, relates to #42968 

Signed-off-by: CrazyMax <crazy-max@users.noreply.github.com>

### CODE REVIEW

File: vendor.conf [hunk 1]
```diff
@@ -119,15 +120,15 @@ github.com/bsphere/le_go                            7a984a84b5492ae539b79b62fb4a
...
+google.golang.org/api                               c24765c18bb761c90df819dcdfdd62f9a7f6fa22 # v0.20.0
 github.com/golang/groupcache                        869f871628b6baa9cfbc11732cdf6546b17c1298
 go.opencensus.io                                    d835ff86be02193d324330acdb7d65546b05f814 # v0.22.3
 cloud.google.com/go                                 ceeb313ad77b789a7fa5287b36a1d127b69b7093 # v0.44.3
 github.com/googleapis/gax-go                        bd5b16380fd03dc758d11cef74ba2e3bc8b0e8c2 # v2.0.5
 google.golang.org/genproto                          3f1135a288c9a07e340ae8ba4cc6c7065a3160e8
 
 # containerd
-github.com/containerd/containerd                    72cec4be58a9eb6b2910f5d10f1c01ca47d231c0 # v1.5.5
+github.com/containerd/containerd                    ee27cde735e2a27a5e1465116ac98593b78d0a6f # v0.0.0-20210721160646-ee27cde735e2
```
Comment (thaJeztah) [inline vendor.conf:131]: Should we consider updating to `v1.6.0-beta.1` ? https://github.com/containerd/containerd/compare/ee27cde735e2a27a5e1465116ac98593b78d0a6f..v1.6.0-beta.1

File: vendor/github.com/containerd/containerd/go.mod [hunk 2]
```diff
@@ -3,7 +3,7 @@ module github.com/containerd/containerd
 go 1.16
 
 require (
-	github.com/Microsoft/go-winio v0.4.17
+	github.com/Microsoft/go-winio v0.5.0
```
Comment (thaJeztah) [inline vendor/github.com/containerd/containerd/go.mod:6]: Haven't checked yet, but any other (indirect) dependency from containerd that needs updating?

File: vendor.conf (hunk 1 above)
Comment (crazy-max) [reply vendor.conf:131]: Yes that's the idea, was just doing some tests in this PR to repro with only containerd from #42968.

File: vendor/github.com/containerd/containerd/go.mod (hunk 2 above)
Comment (crazy-max) [reply vendor/github.com/containerd/containerd/go.mod:6]: Yes we are already on 0.5.1: https://github.com/moby/moby/blob/229bc1ee8bd3939634a5eb3bfebfd59aed399e2d/vendor.conf#L3

File: vendor.conf [hunk 3]
```diff
@@ -18,8 +18,8 @@ github.com/moby/sys                                 9b0136d132d8e0d1c116a38d7ec9
 github.com/creack/pty                               2a38352e8b4d7ab6c336eef107e42a55e72e7fbc # v1.1.11
 github.com/sirupsen/logrus                          bdc0db8ead3853c56b7cd1ac2ba4e11b47d7da6b # v1.8.1
 github.com/tchap/go-patricia                        a7f0089c6f496e8e70402f61733606daa326cac5 # v2.3.0
-golang.org/x/net                                    e18ecbb051101a46fc263334b127c89bc7bff7ea
-golang.org/x/sys                                    63515b42dcdf9544f4e6a02fd7632793fde2f72d
+golang.org/x/net                                    e18ecbb051101a46fc263334b127c89bc7bff7ea # v0.0.0-20210226172049-e18ecbb05110
+golang.org/x/sys                                    63515b42dcdf9544f4e6a02fd7632793fde2f72d # v0.0.0-20210823070655-63515b42dcdf
```
Comment (crazy-max) [inline vendor.conf:22]: Just to show the pseudo ver with timestamp to sort out deps easily.

**Comment by crazy-max** [comment]
```text
errors in package: [/go/src/github.com/docker/docker/builder/builder-next/worker/worker.go:36:2: could not import github.com/moby/buildkit/solver/llbsolver/ops (/go/src/github.com/docker/docker/vendor/github.com/moby/buildkit/solver/llbsolver/ops/build.go:14:2: could not import github.com/moby/buildkit/solver/llbsolver (/go/src/github.com/docker/docker/vendor/github.com/moby/buildkit/solver/llbsolver/result.go:8:2: could not import github.com/moby/buildkit/cache/contenthash (/go/src/github.com/docker/docker/vendor/github.com/moby/buildkit/cache/contenthash/checksum.go:512:19: Seeker not declared by package iradix))) /go/src/github.com/docker/docker/builder/builder-next/worker/worker.go:41:2: could not import github.com/moby/buildkit/source/local (/go/src/github.com/docker/docker/vendor/github.com/moby/buildkit/source/local/local.go:11:2: could not import github.com/moby/buildkit/cache/contenthash (/go/src/github.com/docker/docker/vendor/github.com/moby/buildkit/cache/contenthash/checksum.go:512:19: Seeker not declared by package iradix))]
```

@thaJeztah Ok it seems I can't update containerd without updating buildkit so I guess I will continue working on #42968

**Comment by crazy-max** [comment]
Ok one step further and got the same issue as #42968:

```
2021-11-02T23:35:14.954Z] === Failed

[2021-11-02T23:35:14.954Z] === FAIL: arm64.integration.service TestCreateServiceSecretFileMode (12.14s)

[2021-11-02T23:35:14.954Z]     create_test.go:294: assertion failed: string "\x03\x00\x00\x00\x00\x00\x00\xc2Error grabbing logs: rpc error: code = Unknown desc = warning: incomplete log stream. some logs could not be retrieved for the following reasons: node 00j7mzrtycycgdzqlwwrtkpzv is not available\n" does not contain "-rwxrwxrwx"

[2021-11-02T23:35:14.954Z] 

[2021-11-02T23:35:14.954Z] === FAIL: arm64.integration.service TestCreateServiceConfigFileMode (12.34s)

[2021-11-02T23:35:14.954Z]     create_test.go:351: assertion failed: string "\x03\x00\x00\x00\x00\x00\x00\xc2Error grabbing logs: rpc error: code = Unknown desc = warning: incomplete log stream. some logs could not be retrieved for the following reasons: node 3x6nd25lrct2yh3yvqvcr2ufq is not available\n" does not contain "-rwxrwxrwx"
```

**Comment by tonistiigi** [comment]
cc @AkihiroSuda , possible regression in 1.6

**Comment by crazy-max** [comment]
Ok after just updating grpc I've got the same issue so not linked to containerd itself. Found out [this comment](https://github.com/docker/swarmkit/blob/7956265ead9931dcea6357415d75080e95d9e7ed/vendor.conf#L3-L10) on swarmkit repo from @dperny and @cyli and was wondering if it could be linked? Tried to also [upgrade grpc on swarmkit](https://github.com/crazy-max/swarmkit/commit/b91d9b48802f25694d6e68c278ac0532e471af76) and regen protobuf but no changes.

**Comment by thaJeztah** [comment]
superseded by https://github.com/moby/moby/pull/43357, which was merged

**Pull request closed without merging**